lib/rouge/demos/livescript

```diff
@@ -0,0 +1,1441 @@
+/*
+a multiline comment
```

Comment:
Please make the demo less than 20 lines. This is intended to be a kind of "Hello world" sample to show on the website, as opposed to the comprehensive stress-test that is in `spec/visual`.

Thanks!